apps/frontend/src/libs/components/header/libs/components/notifications-popover/notifications-popover.tsx

```diff
@@ -59,27 +59,29 @@ const NotificationsPopover = ({
 	}, [isOpened, onPageReset]);
 
 	const hasNotifications = notifications.length !== EMPTY_LENGTH;
-	const isLoadingMore = hasNextPage && dataStatus === DataStatus.PENDING;
+	const isLoadingMore = dataStatus === DataStatus.PENDING;
+	const shouldShowPlaceholder = !hasNotifications && !isLoadingMore;
 
 	return (
 		<Popover
 			content={
 				<div className={styles["notifications-popover"]}>
 					<h3 className={styles["notifications-title"]}>Notifications</h3>
 					<div className={styles["notifications"]}>
-						{hasNotifications ? (
-							notifications.map((notification) => (
-								<NotificationItem
-									key={notification.id}
-									message={notification.payload}
-									timestamp={formatRelativeTime(notification.createdAt)}
-								/>
-							))
-						) : (
+						{hasNotifications
+							? notifications.map((notification) => (
+									<NotificationItem
+										key={notification.id}
+										message={notification.payload}
+										timestamp={formatRelativeTime(notification.createdAt)}
+									/>
+								))
+							: null}
```

Comment:
To render nothing we are using this approach
```suggestion
							: <></>}
```